collectors/ebpf_process.plugin/ebpf_process.c

```diff
@@ -213,9 +213,9 @@ static inline void netdata_write_chart_cmd(char *type
             , order);
 }
 
-static void netdata_write_global_dimension(char *dimension, char *name)
+static void netdata_write_global_dimension(char *d, char *n)
 {
-    printf("DIMENSION %s %s absolute 1 1\n", dimension, name);
+    printf("DIMENSION %s %s absolute 1 1\n", d, n);
```

Comment:
Why haven't you changed
https://github.com/netdata/netdata/blob/d00b165c3affded68a984ddc42d73413d0835d1d/collectors/ebpf_process.plugin/ebpf_process.h#L52-L53
https://github.com/netdata/netdata/blob/d00b165c3affded68a984ddc42d73413d0835d1d/collectors/ebpf_process.plugin/ebpf_process.c#L620-L621
https://github.com/netdata/netdata/blob/d00b165c3affded68a984ddc42d73413d0835d1d/collectors/ebpf_process.plugin/ebpf_process.c#L227
instead? Short names just hid the problem, not solved it.

Reply:
When I changed was too late, and I classified this as a less important thing, considering that this is not a bug, the pointers are NULL terminated and the charts have been generated. 
I think I can create a new PR renaming variables late.